pkg/cloudvolumes/get.go

```diff
@@ -0,0 +1,123 @@
+// (C) Copyright 2019 Hewlett Packard Enterprise Development LP.
+
+package cloudvolumes
+
+import (
+	"fmt"
+	"github.com/HewlettPackard/hpecli/pkg/logger"
+	"github.com/HewlettPackard/hpecli/pkg/store"
+	"github.com/spf13/cobra"
+
+	"net/http"
+	"io/ioutil"
+
+	"crypto/tls"
+	"encoding/base64"
+)
+
+var (
+	path string
+	jsonresults bool
+)
+
+func init() {
+	cmdCloudVolumesGet.Flags().StringVar(&host, "host", "", "CloudVolumes portal")
+	cmdCloudVolumesGet.Flags().StringVar(&path, "path", "", "resource to get")
+	cmdCloudVolumesGet.MarkFlagRequired("path")
+	cmdCloudVolumesGet.Flags().BoolVar(&jsonresults, "json", false, "display result in json")
+
+}
+
+// getCmd represents the get command
+var cmdCloudVolumesGet= &cobra.Command{
+	Use:   "get",
+	Short: "Get from Cloud Volumes: hpecli cloudvolumes get",
+	RunE:  runGet,
+}
+
+func runGet(cmd *cobra.Command, args []string) error {
+	fmt.Println("cloudvolumes/get called")
```

Comment:
There are a number of these in this file.  why are these not going through the logger?